server/test/list_objects.go

```diff
@@ -0,0 +1,203 @@
+package test
+
+import (
+	"context"
+	"io/ioutil"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/openfga/openfga/pkg/id"
+	"github.com/openfga/openfga/pkg/logger"
+	"github.com/openfga/openfga/pkg/telemetry"
+	"github.com/openfga/openfga/pkg/testutils"
+	"github.com/openfga/openfga/server/commands"
+	serverErrors "github.com/openfga/openfga/server/errors"
+	"github.com/openfga/openfga/storage"
+	"github.com/stretchr/testify/require"
+	openfgapb "go.buf.build/openfga/go/openfga/api/openfga/v1"
+	"google.golang.org/protobuf/encoding/protojson"
+)
+
+const (
+	defaultListObjectsDeadline   = 5 * time.Second
+	defaultListObjectsMaxResults = 5
+)
+
+type listObjectsTestCase struct {
+	_name          string
+	request        *openfgapb.ListObjectsRequest
+	expectedError  error
+	expectedResult []string //all the results. the server may return less
+}
+
+var tKAllAdminsRepo6 = &openfgapb.TupleKey{
```

Comment:
Let's add a helper/utility function somewhere in `pkg` to streamline these instantiations. 

```
func tuplekey(object, relation, user string) *openfgav1.TupleKey {
	return &openfgav1.TupleKey{
		Object:   object,
		Relation: relation,
		User:     user,
	}
}
```

then these simply become, for example:

```
var tkAllAdminsRepo6 = tuplekey("repo:6", "admin", "*")
```